packages/flutter_test/lib/src/controller.dart

```diff
@@ -338,6 +338,17 @@ abstract class WidgetController {
   ///
   /// The `speed` is in pixels per second in the direction given by `offset`.
   ///
+  /// The `offset` represents the motion one's finger would perform on the
```

Comment:
Can you make this a template for re-use?

Reply:
@Piinks it is slightly different in the cases and it is already _inside_ of another template. Can you nest templates? Otherwise, this is impossible.